eligibility_api/client.py

```diff
@@ -139,28 +139,36 @@ def __init__(
 class Client:
     """Eligibility Verification API HTTP client."""
 
-    def __init__(self, agency, verifier, issuer):
-        logger.debug(f"Initialize client for agency: {agency.short_name}")
-        self.issuer = issuer
-
-        # get the eligibility type names
-        self.types = list(map(lambda t: t.name, agency.types_to_verify()))
-        self.agency_id = agency.agency_id
-        self.jws_signing_alg = agency.jws_signing_alg
-        self.client_private_jwk = agency.private_jwk
-        self.jwe_encryption_alg = verifier.jwe_encryption_alg
-        self.jwe_cek_enc = verifier.jwe_cek_enc
-        self.server_public_jwk = verifier.public_jwk
-
-        self.api_url = verifier.api_url
-        self.api_auth_header = verifier.api_auth_header
-        self.api_auth_key = verifier.api_auth_key
+    def __init__(
+        self,
+        api_url,
```

Comment:
consider renaming to `verify_url` - I'm not sure though? The exact endpoint is not defined by the spec so this isn't a base URL to the server, it is the absolute verify URL.